operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/source/informer/Mappers.java

```diff
@@ -83,9 +83,10 @@ public static <T extends HasMetadata> SecondaryToPrimaryMapper<T> fromOwnerRefer
 
   public static <T extends HasMetadata> SecondaryToPrimaryMapper<T> fromOwnerReferences(
       String apiVersion, String kind, boolean clusterScope) {
+    String correctApiVersion = apiVersion.startsWith("/") ? apiVersion.substring(1) : apiVersion;
```

Comment:
This is probably not the right way to fix the problem. Maybe `HasMetadata.getApiVersion` should just return "v1"?

Reply:
Yes, this seems to be a bug in fabric8 client, pls create an issue also there.

Reply:
I created a PR for fabric8: https://github.com/fabric8io/kubernetes-client/pull/6941
Feel free to propose another fix if it is not what you were thinking.